serverless.yml

```diff
@@ -50,19 +48,32 @@ functions:
               - Arn
 
   handleCommand:
-    handler: stopcovid.command_stream.handle_command
+    handler: stopcovid/command_stream.handle_command
     environment:
       DIALOG_TABLE_NAME_SUFFIX: ${self:provider.stage}
       REGISTRATION_VALIDATION_URL: https://eslworks-api-production.herokuapp.com/slowcovid/lookup
       REGISTRATION_VALIDATION_KEY: ${env:REGISTRATION_VALIDATION_KEY}
 
 
   logMessage:
-    handler: stopcovid.message_log.log_message
+    handler: stopcovid/message_log.log_message
     timeout: 20  # Give aurora cluster time to cold start
 
+  distributeDialogEvents:
+    handler: stopcovid/event_distributor.distribute_dialog_events
+    events:
+      - stream:
+          type: dynamodb
+          arn:
+            Fn::GetAtt:
+              - DialogEvents
+              - StreamArn
+          batchSize: 3
+          batchWindow: 1
```

Comment:
any reason why we shouldn't leave these unspecified and rely on the defaults? Appears that the default is to call the lambda ASAP. https://docs.aws.amazon.com/lambda/latest/dg/with-ddb.html

Reply:
Let me look into this. When I first set up the stream it wasn't hitting the lambda until I added the batching definitions. It's possible something else was wrong, though

Reply:
You're right -- removing the definitions gave us the behavior we want